calico.go

```diff
@@ -178,6 +178,7 @@ func cmdAdd(args *skel.CmdArgs) error {
 			endpoint = api.NewWorkloadEndpoint()
 			endpoint.Metadata.Name = args.IfName
 			endpoint.Metadata.Node = nodename
+			endpoint.Metadata.ActiveInstanceID = args.ContainerID
```

Comment:
The reason why I am thinking this is what if some other orchestrator decides to be "aggressive" with DELs in future, we should be populating `ActiveInstanceID` similarly across all orchestrators so it's not confusing when someone looks at the code. Another reason is we go through different code paths for ADD (for k8s and non-k8s), but same code path for DEL and DEL code checks for `ActiveInstanceID` to avoid double-delete situation, which is unique to k8s for now, but could arise in future for any other orchestrators we support